docs/guides/getting-started-java-spring.md

```diff
@@ -0,0 +1,249 @@
+   </repository>
+</repositories>
+```
+
+2. Add the following dependency to your `pom.xml` file:
+
+```
+<dependency>
+   <groupId>io.camunda</groupId>
+   <artifactId>spring-boot-starter-camunda-sdk</artifactId>
+   <version>8.5.0</version>
+</dependency>
+```
+
+### Configure the Zeebe client
+
+Open your `src/main/resources/application.properties` file, and paste the following snippet to connect to the Self-Managed Zeebe broker:
+
+```
+zeebe.client.broker.grpcAddress=http://127.0.0.1:26500
+zeebe.client.broker.restAddress=http://127.0.0.1:8080
+zeebe.client.security.plaintext=true
+```
+
+### Create a worker
+
+1. In `src/main/java/io/camunda/demo/process_payments/`, create a file called `ChargeCreditCardWorker.java`.
+2. In the file created above, paste the following dependencies and package `package io.camunda.demo.process_payments`:
+
+```java
+package io.camunda.demo.process_payments;
+
+import java.util.Map;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+import io.camunda.zeebe.spring.client.annotation.JobWorker;
+import io.camunda.zeebe.spring.client.annotation.Variable;
+```
+
+3. Next, we can decorate our class with `@Component` and instantiate a logger. Additionally, we will add a method and decorate the `chargeCreditCard` method with `@JobWorker`, and specify a `@Variable(name = "totalWithTax") Double totalWithTax` argument into the `chargeCreditCard` method we'll implement:
+
+```java
+@Component
+public class ChargeCreditCardWorker {
+
+// Instantiate a logger
+  private final static Logger LOG = LoggerFactory.getLogger(ChargeCreditCardWorker.class);
+
+// Add the following method, and decorate `chargeCreditCard` with `@JobWorker`
+  @JobWorker(type = "charge-credit-card")
+
+// Specify a `@Variable(name = "totalWithTax") Double totalWithTax` argument into the `chargeCreditCard` method
+  public Map<String, Double> chargeCreditCard(@Variable(name = "totalWithTax") Double totalWithTax) {
+
+   // Implement the method
+    LOG.info("charging credit card: {}", totalWithTax);
+
+    return Map.of("amountCharged", totalWithTax);
+  }
+}
+```
+
+:::note
+To check your work, visit our [sample repository](https://github.com/camunda/camunda-8-get-started-spring/blob/main/src/main/java/io/camunda/demo/process_payments/ChargeCreditCardWorker.java) with the completed code.
+:::
+
+In your terminal, run `mvn spring-boot:run`, where you should see the `charging credit card` output. In Operate, refresh if needed, and note the payment has executed.
+
+## Step 7: Start a process instance
+
+To start a process instance programmatically, take the following steps:
+
+1. In `ProcessPaymentsApplication.java`, instantiate a static logger and declare `zeebeClient` with the `@Autowired` annotation. To convert the application to a `CommandLineRunner`, add `implements CommandLineRunner` to the class declaration `public class ProcessPaymentsApplication implements CommandLineRunner`.
```

Comment:
```suggestion
1. In `ProcessPaymentsApplication.java`, convert the application to a `CommandLineRunner`, by adding `implements CommandLineRunner` to the `ProcessPaymentsApplication` class declaration. Instantiate a static `Logger` variable, and an instance variable named `zeebeClient` with the `@Autowired` annotation. 
```

Similar to previous comments -- adds a bit of precision and accuracy, in the order in which the code is written.

Reply:
Resolved in latest commit 👍